### PR TITLE
[#710] Update color palette

### LINKED ISSUE #710: Update color palette
See attached file for colors

Update the color palette, see zeplin for up to date colors

### PR DESCRIPTION
**Pull request by planktonic**
Fixes #710 

## Changes

- Updates color design tokens to the new colors, taken from the styleguide in zeplin
- Color variations now use the naming format `light-4`, `light-3`, `light-2`, `light-1`, `base`, `dark-1`, `dark-2`, `dark-3`, `dark-4`, instead of `lightest`, `lighter`, `light`, `base`, `dark`, `darker`, `darkest`. Note that there’re new points on the scale — `dark-4` and `light-4` did not exist in the old scale.
- Updates color usage throughout the codebase

## How to QA

Pull this branch, and run `yarn storybook`.

Most changes in this PR are updating the use of `u-fg`, `u-bg`, and `u-border` utility classes. The core changes are in `settings/_color-palette.scss`.

Check the color-palette page in storybook, and click around some components.

⚠️ the components may not look beautiful — updating them to properly use these colors will come in future PRs, when we update the base styles and component styles to match the new designs (see zeplin).

## 📸

design tokens color palette page

## Checks

Delete if not applicable:

- [x] Storybook documentation has been updated
- [x] Fixtures in [`test/scss/`](../test/scss/) have been updated
- [x] Your changes have been added to the `unreleased` section of [CHANGELOG.md](../CHANGELOG.md)

### CODE REVIEW

File: scss/bitstyles/settings/color-palette.stories.mdx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
 
 # Color palette
 
-Bitstyles provides a semantically-named color palette, with each color having named variants such as `dark`, `light`, `base`. These are used to create the components and the color-related utility classes.
+Bitstyles provides a semantically-named color palette, with each color having a scale of named variants from `dark-3` to `light-3`. These colors are used to create the components and the color-related utility classes like background-color, border, and color. A `color-palette.get()` helper function makes using them in your projects easy.
```
Comment (ed-eva) [inline scss/bitstyles/settings/color-palette.stories.mdx:7]:
```suggestion
Bitstyles provides a semantically-named color palette, with each color having a scale of named variants from `dark-4` to `light-4`. These colors are used to create the components and the color-related utility classes like background-color, border, and color. A `color-palette.get()` helper function makes using them in your projects easy.
```

File: scss/bitstyles/settings/color-palette.stories.mdx [hunk 2]
```diff
@@ -48,16 +48,16 @@ Use the appropriate utility classes to apply colors. See the documentation of ea
 The color palette is also available as CSS custom properties. In the default configuration, they are namespaced with `bs`, see [setup](/story/design-tokens-setup--page):
 
 ```css
---bs-color-brand-1-lighter: #f4f2ff;
---bs-color-brand-1-lighter-rgb: 244, 242, 255;
+--bs-color-brand-1-light-3: #f4f2ff;
```
Comment (ed-eva) [inline scss/bitstyles/settings/color-palette.stories.mdx:51]:
```suggestion
--bs-color-brand-1-light-3: #ded9ff;
```

File: scss/bitstyles/settings/color-palette.stories.mdx [hunk 3]
```diff
@@ -48,16 +48,16 @@ Use the appropriate utility classes to apply colors. See the documentation of ea
...
+--bs-color-brand-1-light-3-rgb: 244, 242, 255;
```
Comment (ed-eva) [inline scss/bitstyles/settings/color-palette.stories.mdx:52]:
```suggestion
--bs-color-brand-1-light-3-rgb: 222, 217, 255;
```

File: scss/bitstyles/settings/color-palette.stories.mdx [hunk 4]
```diff
@@ -48,16 +48,16 @@ Use the appropriate utility classes to apply colors. See the documentation of ea
...
 --bs-color-brand-1-light: #c8bfff;
```
Comment (ed-eva) [inline scss/bitstyles/settings/color-palette.stories.mdx:53]:
```suggestion
--bs-color-brand-1-light-1: #b2a6ff;
```

File: scss/bitstyles/settings/color-palette.stories.mdx [hunk 5]
```diff
@@ -48,16 +48,16 @@ Use the appropriate utility classes to apply colors. See the documentation of ea
...
---bs-color-brand-1-light-rgb: 200, 191, 255;
+--bs-color-brand-1-light-1-rgb: 200, 191, 255;
```
Comment (ed-eva) [inline scss/bitstyles/settings/color-palette.stories.mdx:54]:
```suggestion
--bs-color-brand-1-light-1-rgb: 178, 166, 255;
```

File: scss/bitstyles/settings/color-palette.stories.mdx [hunk 6]
```diff
@@ -48,16 +48,16 @@ Use the appropriate utility classes to apply colors. See the documentation of ea
...
 --bs-color-brand-1-base: #644dff;
```
Comment (ed-eva) [inline scss/bitstyles/settings/color-palette.stories.mdx:55]:
```suggestion
--bs-color-brand-1-base: #9485ff;
```

File: scss/bitstyles/settings/color-palette.stories.mdx [hunk 7]
```diff
@@ -48,16 +48,16 @@ Use the appropriate utility classes to apply colors. See the documentation of ea
...
 --bs-color-brand-1-base-rgb: 100, 77, 255;
 // …
```
Comment (ed-eva) [inline scss/bitstyles/settings/color-palette.stories.mdx:57]:
```suggestion
--bs-color-brand-1-base-rgb: 148, 133, 255;
// …
```

File: scss/bitstyles/settings/color-palette.stories.mdx [hunk 8]
```diff
@@ -66,7 +66,7 @@ The color design tokens can be imported and used in JS:
 ```js
 import colorPalette from '../exports/_color-palette.scss';
 
-console.log(colorPalette['brand-1.lighter']); // '#f4f2ff'
+console.log(colorPalette['brand-1.light-3']); // '#f4f2ff'
```
Comment (ed-eva) [inline scss/bitstyles/settings/color-palette.stories.mdx:69]:
```suggestion
console.log(colorPalette['brand-1.light-3']); // '#ded9ff'
```

File: scss/bitstyles/settings/color-palette.stories.mdx [hunk 9]
```diff
@@ -66,7 +66,7 @@ The color design tokens can be imported and used in JS:
...
 console.log(colorPalette['brand-1.light']); // '#c8bfff'
```
Comment (ed-eva) [inline scss/bitstyles/settings/color-palette.stories.mdx:70]:
```suggestion
console.log(colorPalette['brand-1.light']); // '#b2a6ff'
```

File: scss/bitstyles/settings/color-palette.stories.mdx [hunk 10]
```diff
@@ -66,7 +66,7 @@ The color design tokens can be imported and used in JS:
...
 console.log(colorPalette['brand-1.base']); // '#644dff'
```
Comment (ed-eva) [inline scss/bitstyles/settings/color-palette.stories.mdx:71]:
```suggestion
console.log(colorPalette['brand-1.base']); // '#9485ff'
```

**Review by ed-eva — COMMENTED** [review]
This is great! 
But...
Just a couple hex code changes to match up again

**Comment by planktonic** [comment]
@ed-eva thanks for the review 🙏 I applied all your suggestions

**Review by ArinzeJeffrey-droid — APPROVED** [review]

**Pull request merged**